### PR TITLE
doc: update CONTRIBUTING.md, add stand-alone mode section.

### PR DESCRIPTION
**Pull request by coocood**
Fixes https://github.com/pingcap/tidb/issues/5018

### CODE REVIEW

**Comment by coocood** [comment]
@zhexuany 
PTAL

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -145,31 +145,50 @@ Branch from master:
...
+
+```sh
+# Build the binary.
+make server
+
+# Run in stand-alone mode. The data is stored in `/tmp/tidb`.
+bin/tidb-server
+```
+
+Then you can connect to TiDB with mysql client in another session.
```
Comment (shenli) [inline CONTRIBUTING.md]: What mean by `in another session`?

Comment (coocood) [reply CONTRIBUTING.md]: another terminal

Comment (shenli) [reply CONTRIBUTING.md]: "Then you can connect to TiDB with mysql client." is enough.

**Comment by shenli** [comment]
@queenypingcap PTAL

**Pull request merged**